README.md

```diff
@@ -654,9 +1296,35 @@ import { DocumentView, Config } from 'react-native-pdftron';
 ```
 
 #### Methods
```

Comment:
This will also be hard to maintain. Would be better to group the methods into sections.